### PR TITLE
Pass email address as query param to the `manage account` URL - closes #835

### LINKED ISSUE #835: Append email address to all authed API calls
Helps prompt FXA to pre-load the correct email address.

Companion issue on guardian website https://github.com/mozilla-services/guardian-website/issues/1107

### PR DESCRIPTION
**Pull request by bakulf**

### CODE REVIEW

File: src/urlopener.cpp [hunk 1]
```diff
@@ -5,16 +5,27 @@
...
 }
 
 // static
-void UrlOpener::open(const QUrl& url) {
+void UrlOpener::open(QUrl url, bool addEmailAddress) {
+  if (addEmailAddress) {
+    SettingsHolder* settingsHolder = SettingsHolder::instance();
+    if (settingsHolder->hasUserEmail()) {
+      QUrlQuery query(url.query());
+      query.addQueryItem("email", settingsHolder->userEmail());
```
Comment (birdsarah) [inline src/urlopener.cpp:24]: On guardian website we uriencode the email address. Do the same here?

**Review by birdsarah — CHANGES_REQUESTED** [review]
Add a test for the new codepath of adding email address?

**Review by birdsarah — APPROVED** [review]

**Comment by birdsarah** [comment]
Approved but feel free to make the additions.

File: src/urlopener.cpp (hunk 1 above)
Comment (bakulf) [reply src/urlopener.cpp:24]: Yes.

**Pull request merged**